src/components/AirTicketsSearch/AirTicketsSearch.component.js

```diff
@@ -0,0 +1,187 @@
+import React, { useState, useEffect } from 'react';
+import DateRangePicker from 'react-bootstrap-daterangepicker';
+import 'bootstrap-daterangepicker/daterangepicker.css';
+
+import ItineraryModalContainer from '../ItinerarayModalContainer/ItineraryModalContainer.component';
+import PassengSubContent from '../PassengerSubContent/PassengerSubContent.component';
+
+import {
+	AirTicketsSearchContainer,
+	SearchContainer,
+} from './AirTicketsSearch.styles';
+
+const AirTicketsSearch = () => {
+	const [passengerSubContentHidden, setPassengerSubContentHidden] = useState(
+		true,
+	);
+	const [adultAmount, setAdultAmount] = useState(0);
+	const [childAmount, setChildAmount] = useState(0);
+	const [babyAmount, setBabyAmount] = useState(0);
```

Comment:
```suggestion
	const [passengerAmount, setPassengerAmount] = useState({ adult: 0, child: 0, baby: 0 })
```
이렇게 써줘도 괜찮지 않을까요? 각각의 키값을 업데이트 할 때는 계산된 속성명과 스프레드 연산자 사용할 수 있겠네요

Reply:
음.... 이렇게 하면 passengerAmount(유아 성인 청소년의 총 값을 받는 값)라는 useState 변수가 현재 있는데 변화를 감지할려면 object를 계속 변화 시켜야하는데 한계가 있는 것 같습니다 ㅠㅠ